docs/initial_setup.md

```diff
@@ -0,0 +1,55 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Initial setup
+
+## Run static code analysis locally (recommended)
+
+In order to achieve the best results when using Precaution, it's important to run static code analysis on the codebase in your main branch locally. 
+The reason to do that is that there can be many older source code files which are stable and won't be changed soon by a pull request but still it's good to ensure they don't contain common security-related mistakes.
+
+You can run static code analysis for source code in Python using Bandit: https://github.com/PyCQA/bandit 
+
+and for source code in Go using Gosec: https://github.com/securego/gosec 
+
+## Setting up a Branch Protection Rule
+
+By default, after Precaution is installed, it will not automatically prevent the merging of a pull request even if the check resulted in a failure status. 
+
+![check_fails](./images/check_fails.png)
```

Comment:
Please update this image now that Precaution is not using my profile as a logo.

Reply:
I will.